From 8f1a4e9c42f09350ce4eeccc55150aed46f93bda Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 30 Apr 2009 16:56:54 +0800
Subject: [PATCH] ENGR00112042 Camera viewfinder cannot display correctly on MX25

Fix camera viewfinder display issue due to not set FB_SYNC_CLK_LAT_FALL
in fb_videomode of CPT-VGA case.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 drivers/video/mxc/mxcfb_modedb.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/mxc/mxcfb_modedb.c b/drivers/video/mxc/mxcfb_modedb.c
index 2f0c034..ad31c6b 100644
--- a/drivers/video/mxc/mxcfb_modedb.c
+++ b/drivers/video/mxc/mxcfb_modedb.c
@@ -38,7 +38,7 @@ struct fb_videomode mxcfb_modedb[] = {
 	{
 	 /* 640x480 @ 60 Hz */
 	 "CPT-VGA", 60, 640, 480, 39683, 45, 114, 33, 11, 1, 1,
-	 0,
+	 FB_SYNC_CLK_LAT_FALL,
 	 FB_VMODE_NONINTERLACED,
 	 0,},
 	{
-- 
1.5.4.4

